opencti-platform/opencti-graphql/src/domain/stixDomainObject.js

```diff
@@ -166,7 +182,7 @@ export const stixDomainObjectDelete = async (context, user, stixDomainObjectId)
     throw FunctionalError('Cannot delete the object, Stix-Domain-Object cannot be found.');
   }
   await deleteElementById(context, user, stixDomainObjectId, ABSTRACT_STIX_DOMAIN_OBJECT);
-  notify(BUS_TOPICS[ABSTRACT_STIX_DOMAIN_OBJECT].DELETE_TOPIC, stixDomainObject, user);
+  await notify(BUS_TOPICS[ABSTRACT_STIX_DOMAIN_OBJECT].DELETE_TOPIC, stixDomainObject, user);
```

Comment:
Just curious, why we are waiting in delete function but not in add function above ?

Reply:
Simply an oversight, I will add it

Reply:
I misread the code
The notify returns the complete object, but for the delete method, we only return the id. So we first execute the notify method and the return the id